code/modules/mob/status_procs.dm

```diff
@@ -227,29 +227,43 @@
 /mob/proc/set_disgust(amount)
 	return
 
+/////////////////////////////////// DISABILITIES ////////////////////////////////////
+
+/mob/proc/add_disability(disability, origin)
```

Comment:
All these helpers shouldn't be here if it's living level functionality

Reply:
It'd mean an isliving check every time you try giving a disability. Is it important?

Reply:
I mean, there's a set_disgust that just returns three lines above this

Reply:
This how you get mob_defines.dm

Reply:
Irrelevant vars clutter the vv window, but mob procs add about 10 lines to a single file and save a lot more in other files in typechecking